### DISCUSSION TITLE
Theme not being applied

### DISCUSSION THREAD

**Discussion by simon-pearson** [ohmybash/oh-my-bash#552] [Q&A] [answered]
Hello, I am trying to use oh-my-bash to apply a theme to my terminal. Here's the info about my stack:

* O/S: RHEL 9.3 (Plow) running on a 64-bit system.
* Terminal: Version 3.40.3 for GNOME 40

I have selected the "agnoster" theme in my `.bashrc` file:

```bash
 ✘ simon@localhost  ~  cat ~/.bashrc | grep -i OSH_THEME
OSH_THEME="agnoster"
```

Here's a screenshot of what my terminal looks like, although I have the "agnoster" theme selected:

![image](https://github.com/ohmybash/oh-my-bash/assets/24149735/953bae60-4f88-4c0d-8f95-43e8586a7254)

It feels like something else is interfering and preventing the theme being applied, can anyone help point me in the right direction?

**Comment by akinomyoga** [comment]
As far as I see your picture, the theme appears to be already applied. The prompt in your picture is the one provided by the agnoster theme.

Note that some of the prompt segments are enabled conditionally in the agnoster theme. For example, you will not get the segment showing git information if you are outside any git repository.

**Reply by simon-pearson** [reply]
Perhaps I've misunderstood the theme functionality, I expected applying the "agnoster" theme to also change the background and foreground text colours, as seen in this image:

![image](https://github.com/ohmybash/oh-my-bash/assets/24149735/0cbc2337-eb39-452e-95ef-7af73859464b)

Was this expectation wrong and I need to set the background/foreground colour theme elsewhere?

**Reply by akinomyoga** [reply, marked as answer]
Oh, I see. The background color and the color palette of the terminal are configured by *a theme of the terminal*. Oh My Bash provides *a theme for the shell*, which includes the prompt contents and styles but does not include the terminal's theme.

The way to configure the color theme of the terminal depends on the terminal you use. If you use GNOME Terminal, you can configure the color theme of GNOME Terminal as described in the following pages:

- [Color schemes](https://help.gnome.org/users/gnome-terminal/stable/app-colors.html.en)
- [Change your Linux terminal color theme | Opensource.com](https://opensource.com/article/19/8/add-color-linux-terminal)

You may also search for it by yourself.

**Comment by simon-pearson** [comment]
@akinomyoga, thanks for your help, I see my misunderstanding now.
